### PR TITLE
DAT-2571 Differentiate the size of wiki threshold for Japanese, English and other languages

### PR DESCRIPTION
**Pull request by sebastianmarzjan**

### CODE REVIEW

**Comment by jacekjursza** [comment]
LGTM

File: extensions/wikia/Search/classes/Language/LanguageService.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+	/**
+	 * @return string
+	 */
+	public function getLanguageCode() {
+		return $this->lang;
+	}
+
+	public function getWikiArticlesThreshold() {
+		$resultsLimit = $this->searchLanguageDefinitions[self::ANY_LANG][self::WIKI_ARTICLE_THRESHOLD];
```
Comment (idradm) [inline extensions/wikia/Search/classes/Language/LanguageService.php]: Is this assignment required?

Comment (SebastianMarzjan) [inline extensions/wikia/Search/classes/Language/LanguageService.php]: Yes, either here or in 'else' block below.

File: extensions/wikia/Search/WikiaSearchController.class.php [hunk 2]
```diff
@@ -579,13 +578,15 @@ protected function getSearchConfigFromRequest() {
...
-				$searchConfig->setXwikiArticleThreshold( $threshold );
+
+			$languageService = new \Wikia\Search\Language\LanguageService();
+			$languageService->setLanguageCode( $searchConfig->getLanguageCode() );
+			$wikiArticleThreshold = $languageService->getWikiArticlesThreshold();
+
+			if ( in_array( 'staff', $this->wg->user->getEffectiveGroups() ) ) {
+				$wikiArticleThreshold = $request->getVal( 'minArticleCount', $wikiArticleThreshold );
 			}
+			$searchConfig->setXwikiArticleThreshold( $wikiArticleThreshold );
```
Comment (SebastianMarzjan) [inline extensions/wikia/Search/WikiaSearchController.class.php]: I'm not sure if we should set this if threshold is 0 (filter query), per:
https://github.com/Wikia/app/blob/DAT-2571/extensions/wikia/Search/classes/QueryService/Select/Dismax/InterWiki.php#L117

**Comment by idradm** [comment]
let it be

**Pull request merged**